app/assets/stylesheets/kaui/common.less

```diff
@@ -530,3 +530,24 @@ hr.breathe {
     margin-top: 45px;
     margin-bottom: 45px;
 }
+
+form[id^="new_tag_definition"] {
```

Comment:
It looks like `object_type_line_0` is missing a `padding-bottom`:

![image](https://user-images.githubusercontent.com/51940/34880585-d2c25356-f7a8-11e7-8a53-78ea1eff44fe.png)

Reply:
It should look like this:
![image](https://user-images.githubusercontent.com/10049037/34892737-ca58bac2-f7b0-11e7-91aa-8785fdf92796.png)

Reply:
Maybe cleaning the cache

Reply:
No change after a force reload nor when using an incognito window. I'm using Chrome 63.0.

The behavior is similar on Safari too:

![image](https://user-images.githubusercontent.com/51940/34895712-d46d1002-f7de-11e7-89ac-00a325d9026d.png)

Reply:
This happened to me and I resolve it with `rake tmp:cache:clear`, I do not know if this will work in your case.

Reply:
Oh, you meant Rails caches, not browser caches. Gotcha. Yup, it did the track! 👍